cocos/3d/skeletal-animation/skeletal-animation-utils.ts

```diff
@@ -457,6 +457,11 @@ export interface IAnimInfo {
     buffer: Buffer;
     data: Float32Array;
     dirty: boolean;
+    currentClipRef: WeakSet<AnimationClip>;
```

Comment:
I want to keep a record of `AnimationClip`, something like `WeakRef`. If you have better opinion, please.

Reply:
Just store the clip itself. The whole anim info object will be GCed once the component is destroyed.

Reply:
Got it .